types/api.d.ts

```diff
@@ -18,6 +18,7 @@ type Meta = {
   hasNextPage: boolean
 }
 
+/** @deprecated */
```

Comment:
Would you mind adding a TODO and a ticket to remove this once all the endpoints have been updated?